Sage/carbon
# fix(vertical-divider): change tint type to number

### Proposed behaviour

Change the d.ts file to accept a number (1-100).

### Current behaviour

The `d.ts` file for `VerticalDivider` component expects `tint` to be a `string` but the proptypes and code expect `tint` to be a number.

### Checklist




#### QA

- [x] Tested in sandbox/storybook

### Additional context

Fixes FE-4346, Fixes #4422

### Testing instructions

Ensure there are no TS errors when running.

https://codesandbox.io/s/vertical-divider-ts-xndiq (use codesandbox version below, not this one)

You will need to download this and run it locally due to a codesandbox issue.
